packages/material-ui-lab/src/ToggleButtonGroup/hasValue.js

```diff
@@ -4,5 +4,5 @@ export default function hasValue(value) {
     return value.length > 0;
   }
 
-  return !!value;
+  return value !== undefined;
```

Comment:
For #13488 it would've been sufficient to check against nully via `value != null`. This is closer to how optional PropTypes are checked in react and doesn't introduce a breaking change.

Reply:
so what are you proposing?

Reply:
changed to `!= null`